Marlin/src/pins/stm32f1/pins_CREALITY_V4.h

```diff
@@ -79,6 +79,13 @@
 
 #define Z_PROBE_PIN                         PB1   // BLTouch IN
 
+//
+// Filament Runout Sensor
+//
+#ifndef FIL_RUNOUT_PIN
+  #define FIL_RUNOUT_PIN                    PA4   // "Pulled-high"
+#endif
```

Comment:
Does this mean the Ender 3 V2 example configuration should have `FIL_RUNOUT_STATE LOW` (the default) to indicate that the filament is out? Or, will it depend on the type of sensor installed by the end-user?

Reply:
Note that if the pin is "pulled high" when tested, this probably means the pin is uninitialized and that is the default state. If the default for the STM32 is to pull certain pins high (instead of leaving them floating) then this would make sense.

Reply:
Not sure how this will affect the SD card reader since it is also specified as the SD_CS pin.

Reply:
The `ONBOARD_SD_CS_PIN` is not a named pin that Marlin uses.